dolphinscheduler-dao/src/main/resources/org/apache/dolphinscheduler/dao/mapper/TaskDefinitionLogMapper.xml

```diff
@@ -47,6 +47,9 @@
                 and version = #{item.version})
             </foreach>
         </if>
+        <if test="taskDefinitions == null or taskDefinitions.size == 0">
```

Comment:
Judge taskDefinitions as a required parameter, please add it to the business layer.

Reply:
OK